### DISCUSSION TITLE
Cannot connect to Transmission running as Synology package, but can using same settings with transgui

### DISCUSSION THREAD

**Discussion by lilygthb** [openscopeproject/TrguiNG#90] [Q&A] [answered]
Using [Transmission Remote GUI](https://github.com/transmission-remote-gui/transgui) I can successfully connect to Transmission running on a Synology, not in Docker, but as a Synocommunity Package. I'm using these setting to connect:

<img width="779" alt="2023-10-09H143802@2x" src="https://github.com/openscopeproject/TrguiNG/assets/55871606/05745110-73e1-482c-a7d8-f35ab9e2d6c5">

I can't connect using TrguiNG , I'm using these settings:
<img width="768" alt="2023-10-09H143844@2x" src="https://github.com/openscopeproject/TrguiNG/assets/55871606/ee799c35-7e17-4798-8f0f-9c7b9ee6e1a5">

Which to me look the same settings used by transgui, but I get this error:
<img width="768" alt="2023-10-09H144603@2x" src="https://github.com/openscopeproject/TrguiNG/assets/55871606/9bb56c92-ccc4-4646-ba88-6dc11aa491fe">

Is there something I'm missing?

**Comment by lilygthb** [comment, marked as answer]
I made it work using these settings:

<img width="552" alt="2023-10-09H151424@2x" src="https://github.com/openscopeproject/TrguiNG/assets/55871606/15c1d5ba-0248-42d7-82c2-8ad211ecb5b2">

**Reply by qu1ck** [reply]
Yeah TrguiNG is asking specifically for RPC url which ends with /rpc.
The hint in the field (visible when it's empty) also shows this.
